tests/compatibility_tests/base_test_case/base_compatibility_test.py

```diff
@@ -61,8 +63,11 @@ def load_results_from_file(cls):
     def delete_file(cls):
         """Delete the results file."""
         filepath = cls.get_results_file_path()
-        filepath.unlink()
-        cls.logger.debug(f"Results file deleted: {filepath}")
+        if filepath.exists():
+            filepath.unlink()
+            cls.logger.debug(f"Results file deleted: {filepath}")
```

Comment:
we can run it in debug mode if debugging locally. If these are logged at info level then it would make the logs too verbose and it'd be difficult to find valuable information